public/app/plugins/panel/barchart/types.ts

```diff
@@ -21,7 +21,7 @@ export interface BarChartOptions extends OptionsWithLegend, OptionsWithTooltip,
   groupWidth: number;
   valueRotation: number;
   valueMaxLength: number;
-  rawValue: (seriesIdx: number, valueIdx: number) => number;
```

Comment:
function scan not be on the JSON object :exploding_head: 

(I know this was not introduced in this PR)